cmds/dump.c

```diff
@@ -930,7 +923,9 @@ static void dump_chrome_task_rstack(struct uftrace_dump_ops *ops,
 	}
 
 	/* escape the function name */
-	escape_func_name(name_buf, name);
+	for (i = 0; i < namelen; i++)
+		name_buf_ptr += print_json_escaped_char(name_buf_ptr, &len, name[i]);
+	*name_buf_ptr = '\0';
```

Comment:
I think it should be ok but code analysis tools may concern about the overflow of the name_buf.  We can initialize the len to sizeof(name_buf) - 1 to make them happy.